public/js/authForm.js

```diff
@@ -0,0 +1,4 @@
+window.flip = function(flip) {
```

Comment:
przydałoby się zmodyfikować grunta tak, zeby ostatecznie tworzył tylko jeden plik wynikowy ze wszystkimi skryptami, wtedy do zostałoby skompilowane do tego głównego pliku i zaciągalibyśmy na stronę tylko jeden, ale to dyskusja na później